src/Bundle/BundleMetadata.php

```diff
@@ -23,9 +23,9 @@ class BundleMetadata
     protected $bundle;
 
     /**
-     * @var string|bool
+     * @var string
      */
-    protected $vendor = false;
+    protected $vendor;
```

Comment:
I am for this change but this could be considered a BC break

Reply:
Should be a BC break. The vendor is replaced in every case: https://github.com/sonata-project/SonataEasyExtendsBundle/blob/1954ef9aa2a9aa046d02a914c911bda09120fba7/src/Bundle/BundleMetadata.php#L232

Reply:
This seems contradictory :point_up: